javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime =0;
+    this.intervalId=0;
+    this.intervalId2=0;
+    this.currentMilli=0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(()=>{this.currentTime++}, 1000)
+    this.intervalId2 = setInterval(()=>{this.currentMilli++}, 10)
```

Comment:
There will be a lot of math involved, but instead of doing two setInvervals you could use one and work with it to get the bonus part. Even if you have to break Jasmine heh.